src/eterna/mode/GameMode.ts

```diff
@@ -497,7 +497,7 @@ class ContextMenuDialog extends Dialog<void> {
         super.added();
         this.addObject(this._menu, this.container);
 
-        this._menu.display.position = this._menuLoc;
+        this._menu.display.position.copyFrom(this._menuLoc);
```

Comment:
this is  maybe our only copyFrom (at least so far); have you confirmed we can't use set? hardly an urgent optimization